src/Uno.UI/Microsoft/UI/Xaml/Controls/TeachingTip/TeachingTip.cs

```diff
@@ -0,0 +1,2565 @@
+					break;
+				case TeachingTipPlacementMode.Left:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = LeftEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.Right:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = RightEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.TopLeft:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = TopEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.TopRight:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = TopEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.BottomLeft:
+					templateSettings.TopRightHighlightMargin = BottomLeftPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = BottomLeftPlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.BottomRight:
+					templateSettings.TopRightHighlightMargin = BottomRightPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = BottomRightPlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.LeftTop:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = LeftEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.LeftBottom:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = LeftEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.RightTop:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = RightEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.RightBottom:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = RightEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.Auto:
+					templateSettings.TopRightHighlightMargin = OtherPlacementTopRightHighlightMargin(width, height);
+					templateSettings.TopLeftHighlightMargin = TopEdgePlacementTopLeftHighlightMargin(width, height);
+					break;
+				case TeachingTipPlacementMode.Center:
```

Comment:
Codacy found an issue: [Either merge this case with the identical one on line 694 or change one of the implementations.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6388179)